azure-service-operator.yaml

```
@@ -0,0 +1,71 @@
+package:
+  name: azure-service-operator
+  version: 2.12.0
+  epoch: 0
+  description: Azure Service Operator to provision Azure resources and connect your applications to them from within Kubernetes.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Azure/azure-service-operator
+      tag: v${{package.version}}
+      expected-commit: 5b5d690fd68451d0dcf11896a3122e216c18452d
+
+  - uses: go/bump
+    with:
+      deps: |-
+        golang.org/x/crypto@v0.35.0
+        golang.org/x/net@v0.36.0
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        golang.org/x/oauth2@v0.27.0
+        google.golang.org/protobuf@v1.33.0
+
+  - uses: go/build
+    with:
+      packages: main.go
+      output: ${{package.name}}
```

https://github.com/Azure/azure-service-operator/blob/af07e5eb35ce056be956a444f5c027f984c3a4f3/Dockerfile#L22C73-L22C80

```suggestion
      packages: .
      output: manager
```